docs/ds9.rst

```diff
@@ -17,18 +17,19 @@ representing one DS9 region. The `~regions.Shape` objects can be converted to
     >>> parser = DS9Parser(reg_string)
```

Comment:
Some commits have `[skip ci]` to bypass running the build. Just a reminder that if this goes in the commit message of the HEAD commit in the pull request the CI build will be passed over for this PR which is not what you want. I learned this by having made that mistake some time ago so better to use [skip ci] only for pull requests that do not need CI.

Reply:
Perhaps those fixes in ds9 could go in a separate pull request?

Reply:
Oh! I did not know about that!. I thought it's better to skip ci for minor doc fixes. I will take care of it from next time.
And about those those minor changes in ds9, most of them are related to the changes done in the io/core so I thought it's better to commit in this PR.